instruments/airspeed/__init__.py

```diff
@@ -223,14 +222,17 @@ def __init__(self, parent=None):
 
         # V Speeds
         self.Vs = self.item.get_aux_value('Vs')
+        if self.Vs is None: self.Vs = 0
```

Comment:
There's probably a ton of places that require aux data, and this is just one of them.
It might be better to just bail out and give a nice error message instead of trying to work around and show a low utility display.

Reply:
I'll admit that this is a bit of hackery but Airventure approaches and I really think that the way we get data into these instruments needs to change.  I think that will solve a lot of these types of problems.